.framework/node/backend/lib/event.js

```diff
@@ -12,6 +12,9 @@ const axios = axiosLib.create({
 });
 
 async function sendEvent(event, metadata) {
+  if (process.env.APP_ENV === 'TEST') {
+    return
+  }
```

Comment:
Can we not do that? I prefer to use `nock` or something similar and to not modify code with such ifs